scripts/room.js

```diff
@@ -0,0 +1,43 @@
+// Description:
+//   Sets the permissions for commands in a room
+//
+// Commands:
+//   hubot allow command <command id> - Allows this command in the current room
+//   hubot remove command <command id > - Remove this command in the current room
+
+module.exports = function(robot) {
+
+  robot.respond(/allow command (.*)/i, {id: 'room.allow'}, function(msg) {
+    var plugin = msg.match[1].toLowerCase();
+    var room = msg.message.room;
+
+    var roomPermissions = robot.brain.data.roomPermissions = robot.brain.data.roomPermissions || {};
```

Comment:
Is there a reason this isn't just 

    var roomPermissions = robot.brain.data.roomPermissions || {};